rezasm-app/rezasm-tauri/src/tauri_writer.rs

```diff
@@ -5,17 +5,17 @@ use std::any::Any;
 use std::io::{ErrorKind, Write};
 
 #[derive(Debug)]
-pub struct GuiWriter {}
+pub struct TauriWriter {}
 
-impl GuiWriter {
-    pub fn new() -> GuiWriter {
-        GuiWriter {}
+impl TauriWriter {
+    pub fn new() -> TauriWriter {
+        TauriWriter {}
     }
 }
 
-impl Writer for GuiWriter {}
+impl Writer for TauriWriter {}
 
-impl Write for GuiWriter {
+impl Write for TauriWriter {
```

Comment:
You split this into TauriWriter and WasmWriter. That's sensible, but would t be worth considering making them both into an enum? We did that with the CLI output, having file writing and Console output.

Speaking of which, `OutputSink` is a pretty awful name for that particular structure, but that's a problem for another time.

Reply:
The TauriWriter and WasmWriter are in different packages so I do not see a good way to do this without making things worse. I am going to resolve this comment.